plugin/cmd/main.go

```diff
@@ -123,7 +124,18 @@ func plugin(ctx *cli.Context) {
 		fmt.Println(err)
 		return
 	}
-	clients, err := sdkclients.BuildAll(buildClientConfig, common.HexToAddress(avsConfig.OperatorAddress), signerV2, logger)
+	ecdsaPrivateKey, err := sdkecdsa.ReadKey(avsConfig.EcdsaPrivateKeyStorePath, ecdsaKeyPassword)
+	if err != nil {
+		fmt.Println("Failed to read ecdsa private key")
+		fmt.Println(err)
+		return
+	}
+	clients, err := sdkclients.BuildAll(buildClientConfig, ecdsaPrivateKey, logger)
+	if err != nil {
+		fmt.Println("can't create sdk clients")
```

Comment:
```suggestion
		fmt.Println("can't create sdk clients: %s", err.Error())
```

Reply:
Same for other places

Reply:
Done in [3064cb1](https://github.com/NethermindEth/near-sffl/pull/71/commits/3064cb1e7f1298363b2f6f365c6b25eac729d450)